docs/javascript/v2/how-to-guides/debugging/error-handling.mdx

```diff
@@ -42,10 +42,13 @@ Following are the different error codes that are returned by the SDK. Before ret
 | **3009**       | Device unable to satisfy requested media constraints                                                      | Reduce media constraints for the role in dashboard or upgrade device hardware.                              |
 | **3010**       | No audio detected from track, indicates problem with device hardware                                      | Notify user to try using a headphone for microphone.                                                        |
 | **3011**       | System denied permission to access capture device                                                         | Ask user to check permission granted to audio/video/screen capture devices in system settings.              |
-| **4001**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
-| **4002**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
-| **4003**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
-| **4004**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **3012**       | Current tab is not shared when forceCurrentTab was set to true for screenshare                            | -                                                                                                           |
+| **3013**       | An error occurred while playing audio of remote audio tracks                                               | -                                                                                                           |
+| **3014**       | Selected audio/video device not detected on change                                                        | Ask user to check if the audio/video capture device is connected or not.                                    |
+| **4001**       | WebRTC error                                                                                              | Some webRTC error has occurred. Need more logs to debug.                                                     |
```

Comment:
Let's add the constant name used for the error code here. eg: Create offer failed in this case.

Reply:
Creating a new column for this

Reply:
The 7000 series do not have error strings associated, just added a hyphen there